test/extensions/filters/network/client_ssl_auth/client_ssl_auth_test.cc

```diff
@@ -163,6 +163,10 @@ TEST_F(ClientSslAuthFilterTest, Ssl) {
       std::make_shared<Network::Address::Ipv4Instance>("192.168.1.1"));
   std::string expected_sha_1("digest");
   EXPECT_CALL(*ssl_, sha256PeerCertificateDigest()).WillOnce(ReturnRef(expected_sha_1));
+  EXPECT_CALL(filter_callbacks_.connection_.stream_info_,
+              setResponseFlag(StreamInfo::ResponseFlag::UpstreamProtocolError));
+  EXPECT_CALL(filter_callbacks_.connection_.stream_info_,
+              setResponseCodeDetails("auth_digest_no_match"));
   EXPECT_CALL(filter_callbacks_.connection_, close(Network::ConnectionCloseType::NoFlush));
```

Comment:
@ggreenway `NoFlush` should still be okay right or do we need to change it to `FlushWrite`?

Reply:
`NoFlush` should be fine. Are you seeing an issue with it?

Reply:
No, I am not seeing any issues. Just wanted to confirm once.